doc/hardware/sensors.rst

```diff
@@ -0,0 +1,41 @@
+
+Sensors list
+##################################
+
+.. table:: Supported sensors on MinnowBoard MAX, Galileo GEN 2 and Edison boards
+
+    ============== ============================================= ==========================================================================
+    Sensor Name     Sensor Type                                    Comment
+    ============== ============================================= ==========================================================================
+    BMA180          Accel                                          missing driver on Edison
```

Comment:
these `missing driver on XXX` won't scale. IMO it's better to use a shorter pattern, we can then have a legend afterwards. Like: `-edison` (= no edison) or `=edison` (= only available in edison)